pages/language/index.tsx

```diff
@@ -1,27 +1,104 @@
 import React from "react";
-import { Container, Flex } from "@chakra-ui/react";
-import LanguageIcons from "components/sections/language/LanguageIcons";
+import { Button, VStack, Text, Image } from "@chakra-ui/react";
+import { ArrowForwardIcon } from "@chakra-ui/icons";
```

Comment:
Don't we have a custom icon for this? I think there's an Icon for (Right Icon) `->` in the `Icons` file.

Reply:
resolved 👍